demo/kubernetes/rook-cluster.yaml

```diff
@@ -4,7 +4,28 @@ metadata:
   name: rook
 spec:
   namespace: rook
-  version: latest
-  dataDirHostPath: 
-  useAllDevices: false
-  deviceFilter:
+  version: master-latest
```

Comment:
why is there a specific version tag and not latest? Is it because you havent publish it? If so, shouldnt we?